src/test/java/ApiHandlersConnectionMockitoTests.java

```diff
@@ -0,0 +1,225 @@
+import org.example.ApiServiceConnections.*;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+import static org.mockito.Mockito.lenient;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(MockitoExtension.class)
+public class ApiHandlersConnectionMockitoTests {
+
+    private final String URL = "https://www.dnd5eapi.co/api/classes/";
```

Comment:
isn't this already available in the Constants class?

Reply:
ok